src/components/InputField.vue

```diff
@@ -5,65 +5,72 @@
         <input
           v-model="newItem"
           type="text"
-          placeholder="Enter new Item"
+          :placeholder="$t('input-placeholder')"
           maxlength="50"
         />
         <p class="counter">{{ charcount }}</p>
       </div>
     </div>
 
     <div class="input-form-priority">
-      <label for="id-priority"
-        ><h4 style="margin-right: 1rem">Priority</h4></label
-      >
+      <label for="id-priority">
+        <h4 style="margin-right: 1rem">
+          {{ $t("input-label-priority") }}
+        </h4>
+      </label>
       <label>
         <select id="id-priority" v-model="newItemPriority">
-          <option value="true">High</option>
-          <option value="false">Low</option>
+          <option value="true">{{ $t("input-priority-options.high") }}</option>
+          <option value="false">
+            {{ $t("input-priority-options.low") }}
+          </option>
         </select>
       </label>
     </div>
     <div>
-      <label for="id-item-type"><h4>Item Type</h4></label>
+      <label for="id-item-type">
+        <h4>{{ $t("input-label-item-type") }}</h4>
+      </label>
 
       <label id="id-item-type">
-        <label
-          ><input
-            v-model="newItemType"
-            type="radio"
-            value="true"
-          />Paperback</label
-        >
-        <label
-          ><input v-model="newItemType" type="radio" value="false" />E-Book
+        <label>
+          <input v-model="newItemType" type="radio" value="true" />
+          {{ $t("input-item-type-options.paperback") }}
+        </label>
+
+        <label>
+          <input v-model="newItemType" type="radio" value="false" />
+          {{ $t("input-item-type-options.ebook") }}
         </label>
       </label>
     </div>
     <div class="input-form-bottom">
-      <div>
-        <label for="id-pakegging">
-          <h4>Pakegging <span style="color: red; font-size: 18px">*</span></h4>
-        </label>
+      <label for="id-pakegging">
+        <h4>
+          {{ $t("input-label-packaging") }}
+          <span>*</span>
+        </h4>
+      </label>
 
-        <label id="id-pakegging">
-          <label v-for="i in itemPackageTypeDict" :key="i.id">
-            <input :id="i" type="checkbox" @click="sinkItemPackageType(i)" />
-            {{ i }}
-          </label>
+      <label id="id-pakegging">
+        <label v-for="i in itemPackageTypeDict" :key="i.id">
+          <input :id="i" type="checkbox" @click="sinkItemPackageType(i)" />
+          {{ $t("input-packaging-options.check1") }}
         </label>
-      </div>
+      </label>
     </div>
 
     <div class="save-button-posititon">
       <slot name="footer-btn"></slot>
       <AppButton class="app-p-sm" :disabled="isDisabled" variant="save">
-        Save Item
+        {{ $t("buttons.save-item") }}
       </AppButton>
     </div>
 
     {{ newItem.length }} {{ newItemPackageType }}
```

Comment:
```suggestion
    <span>{{ newItem.length }} </span>
    <span>{{ newItemPackageType }} </span>
```
Usually its good idea to use seperate markup tag for 2 seperate things. inline elements like span do not cause a break in the ui like block elements so user end result is same.

Reply:
I have used this part temporarily to check new item property values. I will remove those when problem will solve